web/package.json

```diff
@@ -28,6 +28,7 @@
     "vite-plugin-wasm-pack": "0.1.11"
   },
   "dependencies": {
+    "@scottish-government/design-system": "^3.0.0-beta.0",
```

Comment:
1) Following https://designsystem.gov.scot/get-started/installation, I didn't see the point of SASS, since we're not customizing their styles. We could always set it up later if needed.

2) I didn't see instructions about bundling in the Roboto font explicitly for SG. Checking the screenshot and inspector mode, it's showing up fine for me on Linux Firefox without anything extra, but I probably have a system font installed somewhere. Do you know if it's necessary to bundle in the font in case people don't have it? (If so, I would do so statically hosting ourselves, not relying on another CDN)

Reply:
I didn't need SASS running it locally.

Roboto should be added as most won't have it locally. Fine to host ourselves.